modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -7,6 +7,34 @@ import (
 	"github.com/cosmos/ibc-go/v6/modules/apps/transfer/types"
 )
 
+func checkEvents(
```

Comment:
But don't have a strong preference, but just thinking that this function could be named `assertEvents` maybe?

Reply:
ahh better name thanks